__docs__/wonder-blocks-tooltip/tooltip.stories.tsx

```diff
@@ -375,23 +375,6 @@ export const AutoUpdate: StoryComponentType = {
         } | null>(null);
         return (
             <View style={[styles.centered, styles.row, {position: "relative"}]}>
-                <Tooltip
-                    content="This is a tooltip that auto-updates its position when the trigger element changes."
-                    opened={true}
-                    autoUpdate={true}
-                >
-                    <View
-                        style={[
-                            position && {
-                                position: "absolute",
-                                top: position.y,
-                                left: position.x,
-                            },
-                        ]}
-                    >
-                        Trigger element
-                    </View>
-                </Tooltip>
```

Comment:
Why was this removed?

Reply:
It was moved to be included after the buttons. See my previous comment here: https://github.com/Khan/wonder-blocks/pull/2373/files#r1862635270